src/cli.rs

```diff
@@ -23,6 +23,13 @@ pub enum Commands {
     Init,
 }
 
+#[derive(ValueEnum, Clone)]
+pub enum SyncTarget {
+    Roblox,
```

Comment:
What do you say we change the names of these targets?

`Roblox` is a little ambiguous to me, considering `Local` goes to Roblox still, but.. Studio. I feel like a better name would be `Cloud` or `Online`.
I also think `Local` could be renamed `Studio`, but that's up for debate, and I'm fine with either.

Reply:
Yeah I agree that it's better if these are changed, those names are more clear